Tests/SynchronizableTests.swift

```diff
@@ -116,32 +116,34 @@ class SynchronizableTests: XCTestCase
             return
         }
         
-        let sharedAccessGroupIdentifier: Identifier
-        #if os(iOS)
-            sharedAccessGroupIdentifier = Identifier(nonEmpty: "com.squareup.Valet-iOS-Test-Host-App")!
-        #elseif os(OSX)
-            sharedAccessGroupIdentifier = Identifier(nonEmpty: "com.squareup.Valet-macOS-Test-Host-App")!
-        #else
-            XCTFail()
-        #endif
-        
-        let permutations: [Valet] = CloudAccessibility.allValues().flatMap { cloudAccessibility in
-            return .sharedAccessGroupValet(with: sharedAccessGroupIdentifier, flavor: .iCloud(cloudAccessibility))
-        }
-        for permutation in permutations {
-            XCTAssertTrue(permutation.canAccessKeychain())
+        for permutation in Valet.iCloudPermutations(Valet.sharedAccessGroupIdentifier, shared: true) {
+            XCTAssertTrue(permutation.canAccessKeychain(), "\(permutation) could not access keychain.")
         }
     }
     
     // MARK: Backwards Compatibility
     
-    func test_backwardsCompatibilityWithObjectiveCValet() {
-        XCTAssert(valet.accessibility == .whenUnlocked)
-        let legacyValet = VALSynchronizableValet(identifier: valet.identifier.description, accessibility: VALLegacyAccessibility.whenUnlocked)!
-        
-        let key = "yo"
-        legacyValet.setString("dawg", forKey: key)
-        
-        XCTAssertEqual(legacyValet.string(forKey: "yo"), valet.string(for: key))
+    func test_backwardsCompatibility_withLegacyValet() {
+        for permutation in Valet.iCloudPermutations(valet.identifier) {
+            let legacyValet = VALSynchronizableValet(identifier: permutation.legacyIdentifier, accessibility: permutation.legacyAccessibility)!
```

Comment:
likely need to guard `testEnvironmentIsSigned`

Reply:
caught this in the next commit :(